nise/report.py

```diff
@@ -69,6 +68,8 @@
 from nise.upload import upload_to_s3
 from nise.util import LOG
 
+# from nise.generators.gcp import CloudStorageGenerator (Functionality to be added)
```

Comment:
I need to delete this.

Reply:
addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff